app/views/place/show.html.erb

```diff
@@ -45,12 +45,12 @@
     <section class="more">
       <div class="further-information">
         <h2 class="govuk-heading-m">Further information</h2>
-        <%= render "govuk_publishing_components/components/govspeak", {} do %>
-          <%= sanitize(@publication.need_to_know) %>
-        <% end %>
-        <%= render "govuk_publishing_components/components/govspeak", {} do %>
-          <%= sanitize(@publication.more_information) %>
-        <% end %>
+        <%= render "govuk_publishing_components/components/govspeak", {
+          content: @publication.need_to_know.html_safe
+        } %>
+        <%= render "govuk_publishing_components/components/govspeak", {
+          content: @publication.more_information.html_safe
+        } %>
       </div>
     </section>
   <% end %>
```

Comment:
Similar story here with the place schema, where seemingly none of the fields are marked as required: https://github.com/alphagov/govuk-content-schemas/blob/7d63f4f503c161e7756a867c02b58af4938dde1b/formats/place.jsonnet#L4-L6 which is really unhelpful.

Reply:
Thanks @kevindew - will fix.